Anytype/Sources/ServiceLayer/SyncStatus/SyncStatusStorage.swift

```diff
@@ -5,21 +5,19 @@ import ProtobufMessages
 
 
 protocol SyncStatusStorageProtocol {
-    func statusPublisher(spaceId: String) -> AnyPublisher<SyncStatusInfo?, Never>
+    func statusPublisher(spaceId: String) async -> AnyPublisher<SyncStatusInfo?, Never>
```

Comment:
How i can subscribe in swiftui screen? Without async i can write:
```
for await status in service.statusPublisher(spaceId: spaceId).values {

}
```

Reply:
<img width="730" alt="image" src="https://github.com/user-attachments/assets/c055172f-f84b-4a47-af87-7c0d867a139c">